Source/rclUE/Private/ROS2Node.cpp

```diff
@@ -133,7 +133,52 @@ void AROS2Node::AddSubscription(const FString TopicName,
     }
 }
 
-void AROS2Node::AddService(const FString ServiceName, const TSubclassOf<UROS2GenericSrv> SrvClass, const FServiceCallback Callback)
+void AROS2Node::AddSubscriptionBP(const FString& InTopicName,
```

Comment:
what is the main intention to intorduce separate AddSubscriptionBP from AddSubscription?

Reply:
As noted in the description and [here](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/pull/30/files#r779359991), we now have all delegates as of `regular` type and only expose ones that require BP specific needs as `dynamic`. 
* Basically, Dynamic delegate could go with `UFUNCTION()` and thus be exposable to BP, but Regular could not :(
* For such reason, certain separate BP-based callback registration APIs need to be added also.
I haven't found a better way to fulfill both cpp and BP's functionalities with one unified API suite.